packages/react-core/src/components/FileUpload/examples/FileUpload.md

```diff
@@ -268,7 +268,8 @@ class CustomFileUpload extends React.Component {
       isLoading: false,
       isDragActive: false,
       hideDefaultPreview: false,
-      children: false
+      children: false,
+      textAreaPlaceholder: false
```

Comment:
Maybe rename this so it is clear it is a boolean.  I was little confused at first since the `textAreaPlaceholder` is a string.  Maybe `hasPlaceholderText`